static/app/types/sampling.tsx

```diff
@@ -102,23 +102,13 @@ export type SamplingRule = {
 
 export type SamplingDistribution = {
   endTimestamp: string | null;
-  null_sample_rate_percentage: null | number;
   project_breakdown:
     | null
     | {
         'count()': number;
         project: string;
         project_id: number;
       }[];
-  sample_rate_distributions: null | {
-    avg: null | number;
-    max: null | number;
-    min: null | number;
-    p50: null | number;
-    p90: null | number;
-    p95: null | number;
-    p99: null | number;
-  };
   sample_size: number;
```

Comment:
can we also remove `sample_size`? we are not using it

Reply:
hmm ngl I feel its useful to have even if it is not being used, just to get an idea of how much the sample size is that the breakdowns are calculated from

Reply:
lets keep it for now, and re-visit later?